packages/botonic-react/src/components/message.jsx

```diff
@@ -154,31 +169,56 @@ export const Message = props => {
 
   const isFromUser = () => from === 'user'
   const isFromBot = () => from === 'bot'
+
   const getBgColor = () => {
     if (!blob) return COLORS.TRANSPARENT
-    return isFromUser()
-      ? getThemeProperty('brand.color', COLORS.BOTONIC_BLUE)
-      : COLORS.SEASHELL_WHITE
+    if (isFromUser()) {
+      const color = getThemeProperty('message.user.style.background', undefined)
+      return color || getThemeProperty('brand.color', COLORS.BOTONIC_BLUE)
+    }
+    const color = getThemeProperty('message.bot.style.background', undefined)
+    return color || COLORS.SEASHELL_WHITE
   }
 
   const getMessageStyle = () =>
     isFromBot()
       ? getThemeProperty('message.bot.style')
       : getThemeProperty('message.user.style')
 
-  const getBlobTick = () => getThemeProperty(`message.${from}.blobTick`, true)
+  const hasBlobTick = () => getThemeProperty(`message.${from}.blobTick`, true)
 
   const renderBrowser = () => {
     const m = webchatState.messagesJSON.find(m => m.id === state.id)
     if (!m || !m.display) return <></>
-    const pointerSize = 6
-    const pointerStyles = {
-      position: 'absolute',
-      top: '50%',
-      width: 0,
-      height: 0,
-      border: `${pointerSize}px solid ${COLORS.TRANSPARENT}`,
-      marginTop: -pointerSize,
+
+    const getBlobTick = pointerSize => {
+      const color =
+        pointerSize == 5
+          ? getBgColor()
+          : getThemeProperty(
+              `message.${from}.style.borderColor`,
+              COLORS.TRANSPARENT
+            )
```

Comment:
This logic it's confusing to me. It's not easy to deduce how this works (a color which depends on the `pointerSize` sounds strange). I suggest you to make this part more readable or at least adding a comment.